src/databricks/labs/lakebridge/assessments/dashboards/dashboard_manager.py

```diff
@@ -0,0 +1,173 @@
+
+        return dashboard
+
+    def create_profiler_summary_dashboard(
+        self,
+        source_tech: str,
+        catalog_name: str = "lakebridge_profiler",
+        schema_name: str = "profiler_runs",
+    ) -> None:
+        """Deploys a profiler summary dashboard to the current Databricks user’s workspace home."""
+
+        logger.info("Deploying profiler summary dashboard.")
+
+        # Load the AI/BI Dashboard template for the source system
+        template_folder = (
+            find_project_root(__file__)
+            / f"src/databricks/labs/lakebridge/resources/assessments/dashboards/{source_tech}"
+        )
+        ws_path = f"/Workspace/Users/{self._current_user}/Lakebridge/Dashboards/"
+        self._create_or_replace_dashboard(
+            folder=template_folder, ws_parent_path=ws_path, dest_catalog=catalog_name, dest_schema=schema_name
+        )
+
+    def upload_duckdb_to_uc_volume(self, local_file_path, volume_path):
+        """
+        Upload a DuckDB file to Unity Catalog Volume
+        Args:
+            local_file_path (str): Local path to the DuckDB file
+            volume_path (str): Target path in UC Volume (e.g., '/Volumes/catalog/schema/volume/myfile.duckdb')
+        Returns:
+            bool: True if successful, False otherwise
+        """
+
+        # Validate inputs
+        if not os.path.exists(local_file_path):
+            logger.error(f"Local file not found: {local_file_path}")
+            return False
+
+        if not volume_path.startswith('/Volumes/'):
+            logger.error("Volume path must start with '/Volumes/'")
+            return False
+
+        try:
+            with open(local_file_path, 'rb') as f:
+                file_bytes = f.read()
+                binary_data = io.BytesIO(file_bytes)
+                self._ws.files.upload(volume_path, binary_data, overwrite=True)
+            logger.info(f"Successfully uploaded {local_file_path} to {volume_path}")
+            return True
+        except FileNotFoundError as e:
+            logger.error(f"Profiler extract file was not found: \n{e}")
+            return False
+        except PermissionDenied as e:
+            logger.error(f"Insufficient privileges detected while accessing Volume path: \n{e}")
+            return False
+        except NotFound as e:
+            logger.error(f"Invalid Volume path provided: \n{e}")
+            return False
+        except InternalError as e:
+            logger.error(f"Internal Databricks error while uploading extract file: \n{e}")
+            return False
+        # Catch any other unexpected exception to avoid crashing the process, but log for debugging.
+        except Exception as e:  # noqa: BLE001
+            logger.error(f"Unexpected error occurred while uploading file: {str(e)}")
+            raise  # Re-raise to avoid swallowing unexpected exceptions (pylint W0718)
```

Comment:
Not sure what this specific raise is intended to do, if we are thinking of failing the cli then you can do 

`System.Exit`